.gitignore

```diff
@@ -12,3 +12,6 @@ node_modules
 
 # Jet Brains
 .idea
+
+# Config
+.env
```

Comment:
Do we need it?

Reply:
Yes, I accidentally committed this file last week. Either I'm using something wrong or it should be in .gitignore

Reply:
It shouldn't be `.env` in the project root :)

Reply:
Where should it be? `dash-network` expects to find it there, otherwise it will generate a blank file

Reply:
Yeah, but `dash-network` is an independent tool that you suppose to run in a directory with configs.

Reply:
So I need to be in the directory with the Dockerfile to build the image, then switch to another directory with configs to run it with `dash-network`? Like this?
```
cd dash-network-deploy
docker build -t strophy/dash-network-deploy:latest
cd networks
../bin/dash-network deploy testnet
cd ..
```

Reply:
As a user you not aupposed to even clone the repository :). You just have configs and you download `dash-network` (see readme) and run it frok the directory with configs (i.e. `dash-network-configs`)